testing/sdk_utils.py

```diff
@@ -165,6 +166,22 @@ def get_metadata() -> requests.Response:
                                    retry=False)
 
 
+def get_cluster_zones():
+    rc, stdout, _ = sdk_cmd.run_cli("node --json")
+
+    if rc != 0:
+        log.warning("return code for command 'node --json' is non-zero: %s", rc)
```

Comment:
`sdk_cmd.run_cli` ultimately already logs out something to that effect, so we can skip on doing this: https://github.com/mesosphere/dcos-commons/blob/4ef375d8c95272041d553c6549f7ed88c21889c1/testing/sdk_cmd.py#L216-L217

Reply:
Removed this check and logging both.

Reply:
Removing the logging was good, but the check is valid. We should keep it.